src/main/java/crawlercommons/sitemaps/sax/XMLIndexHandler.java

```diff
@@ -72,30 +74,24 @@ public void endElement(String uri, String localName, String qName) throws SAXExc
             maybeAddSiteMap();
         } else if ("sitemapindex".equals(currentElement())) {
             sitemap.setProcessed(true);
+        } else if ("loc".equals(currentElement())) {
+            locFound = true;
```

Comment:
No, it's mandatory to set it to true here. `locFound` means "a `<loc>...</loc>` element has been already found in the current `<sitemap>...</sitemap>`". The point is: there are sitemap indexes which just add the URLs without a surrounding`<loc>` (see [sitemap.index.xml](/crawler-commons/crawler-commons/blob/master/src/test/resources/sitemaps/sitemap.index.xml)). Maybe the name `locFound` isn't optimal?